docs/modules/ROOT/pages/how-tos/configuring/rerunning.adoc

```diff
@@ -32,11 +32,14 @@ NOTE: For additional options, refer to the link:https://pipelinesascode.com/docs
 . Comment `/retest` on the commit.
 
 +
-The pipeline run should resume in the *Activity* > *Pipeline runs* tab.
+All pipeline runs associated with that commit should resume in the *Activity* > *Pipeline runs* tab.
 
 +
 NOTE: This procedure only works to retrigger the build pipeline for the latest commit on the branch.
 
++
+NOTE: For instructions on how to retrigger only individual build pipelines, not all, see link:https://pipelinesascode.com/docs/guide/gitops_commands/[pipelinesascode documentation].
```

Comment:
Do you want to direct them specifically to the commands with the `<pipelinerun-name>`?

Reply:
That would be nice, but I don't see an appropriate anchor to link to for that.